CMakeLists.txt

```diff
@@ -96,10 +98,10 @@ else()
   target_link_libraries(rmm INTERFACE CUDA::cudart)
 endif()
 
-target_link_libraries(rmm INTERFACE rmm_logger)
 target_link_libraries(rmm INTERFACE CCCL::CCCL)
 target_link_libraries(rmm INTERFACE dl)
 target_link_libraries(rmm INTERFACE nvtx3::nvtx3-cpp)
+target_link_libraries(rmm INTERFACE rapids_logger::rapids_logger)
```

Comment:
Does the order of `target_link_libraries` matter here?

Reply:
No, the link order here is not important.